www/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Ionic</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+  <link ios-href="build/css/app.ios.css" rel="stylesheet">
+  <link md-href="build/css/app.md.css" rel="stylesheet">
+</head>
+<body>
+
+  <ion-app></ion-app>
```

Comment:
The big difference here is that rather than attaching `ng-app` to the body tag (which would let Ionic know that’s where we want our app to live), we now use `<ion-app>`